tcg_rim_extensions.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2020 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package swid
+
+type AnyURI string
+
+type TcgRimReferenceMeasurementEntry struct {
```

Comment:
Could you please refer to the spec where this extension is defined ?

Reply:
done in c34b2ba